### ISSUE TITLE
[CSS] Enable <color-interpolation-method> on conic, linear, and radial gradients

### ISSUE THREAD

**Issue by dipikabh** [mdn/content#33520]
## Acceptance criteria

- [x] The listed features are documented sufficiently on MDN
- [x] BCD is updated
- [x] Interactive example and data repos are updated if appropriate
- [x] The content has been reviewed as needed

### For folks helping with Firefox related documentation

- [ ] Set bugs to `dev-doc-complete`
- [x] Add entry to Firefox release notes for enabled/preview features
- [x] Add/remove entry to Firefox experimental features page for preference/released features

## Related Gecko bugs

[Bug 1861363](https://bugzilla.mozilla.org/show_bug.cgi?id=1861363): [css-images] Enable `<color-interpolation-method>` on (linear|radial|conical)-gradient

## Issues and Pull Requests

- [x] Relnote: https://github.com/mdn/content/pull/34037
- [ ] Examples: https://github.com/mdn/content/pull/34035
- [x] BCD: https://github.com/mdn/browser-compat-data/pull/23246

## Related pages on MDN

- https://developer.mozilla.org/en-US/docs/Web/CSS/color-interpolation-method
- https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/conic-gradient
- https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/linear-gradient
- https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/radial-gradient
- https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/repeating-conic-gradient
- https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/repeating-linear-gradient
- https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/repeating-radial-gradient

## Other

- [ ] Check content open issues to see if any pertain to the subject matter. If there are any that can be closed because of the work, do so. If there are any that can be fixed relatively quickly because of the knowledge from completing this issue and you have time, feel free to go ahead and fix them.

**Comment by bsmth** [comment]
Waiting on examples to land as a nice-to-have, but everything looks done for this.
